### ISSUE TITLE
Does this work for play.ogarul.tk?

### ISSUE THREAD

**Issue by ghost** [Megabyte918/Agar.io-Macros#1]

**Comment by m-byte918** [comment]
Of course :)

**Comment by segg21** [comment]
Nope. I patched scripts on the site. Your code isn't efficient enough. Better luck next time.

**Comment by freaky767** [comment]
Are you sure?

**Comment by segg21** [comment]
Oh I'm 100% sure, if he was a better developer he would've never coded the way he did. This script is detected, and I will soon start banning people from the client for next year. 

We don't tolerate people who use hack bots and scripts. People who makes them clearly don't have anything else to do with their life but cheat in games. Maybe you can cheat with agar.io, but we aren't agar.io. We are doing things totally different.

**Comment by segg21** [comment]
@freaky767 new security update, this script is 100% patched now,

this issue should be closed due to it no longer working.
